lib/jekyll.py

```diff
@@ -29,7 +29,8 @@
     )
 
 from .files import (
-    open_config,
+    open_read_config,
+    open_write_config,
```

Comment:
@rht `open_config` was present but unused. And `open_read_config` and `open_write_config` were used but not imported. I was facing errors when I was running it.